### PR TITLE
operations: properly close FileOutputStream.

### PR DESCRIPTION
**Pull request by ardevd**
Classes that implement the Closeable interface or its super-interface, AutoCloseable, needs to be closed after use. try-with-resources does this for you automatically.

### CODE REVIEW

File: src/main/java/com/owncloud/android/operations/DownloadFileOperation.java [hunk 1]
```diff
@@ -203,8 +203,9 @@ protected RemoteOperationResult run(OwnCloudClient client) {
                 try {
                     byte[] decryptedBytes = EncryptionUtils.decryptFile(tmpFile, key, iv, authenticationTag);
 
-                    FileOutputStream fileOutputStream = new FileOutputStream(tmpFile);
-                    fileOutputStream.write(decryptedBytes);
+                    try(FileOutputStream fileOutputStream = new FileOutputStream(tmpFile)) {
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/operations/DownloadFileOperation.java]:
```suggestion
                    try (FileOutputStream fileOutputStream = new FileOutputStream(tmpFile)) {
```

**Comment by tobiasKaminsky** [comment]
:+1: tested and works :tada: 
I added the whitespace and squashed lint/drone.

Thanks @ardevd!

File: src/main/java/com/owncloud/android/operations/DownloadFileOperation.java [hunk 2]
```diff
@@ -203,8 +203,9 @@ protected RemoteOperationResult run(OwnCloudClient client) {
                 try {
                     byte[] decryptedBytes = EncryptionUtils.decryptFile(tmpFile, key, iv, authenticationTag);
 
-                    FileOutputStream fileOutputStream = new FileOutputStream(tmpFile);
-                    fileOutputStream.write(decryptedBytes);
+                    try (FileOutputStream fileOutputStream = new FileOutputStream(tmpFile)) {
```
[the remark this answers is not in the document]
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/operations/DownloadFileOperation.java]: This is only available on sdk 26+, so we cannot use this right now.

[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/operations/DownloadFileOperation.java]: correct, so should we deactivate this rule in general to not have it reported all the time? :)

[the remark this answers is not in the document]
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/operations/DownloadFileOperation.java]: Best would be that codacy automatically detects which minSDK we use…
Deactivating will maybe lead to forgetting this, once we can do it.

[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/operations/DownloadFileOperation.java]: Codacy is Java only, not Android

**Pull request merged**